docs/.eslintignore

```diff
@@ -0,0 +1 @@
+src/documentation/06-kiwi-use/02-content/02-grammar-and-mechanics.mdx
```

Comment:
This file somehow always errors out our lint job. It seems counter-productive at this point to try and fix the linting setup just for this file.

Reply:
Yes! Done